packages/react-components/react-tree/src/components/TreeItem/renderTreeItem.tsx

```diff
@@ -15,7 +16,9 @@ export const renderTreeItem_unstable = (state: TreeItemState, contextValues: Tre
         <slots.root {...slotProps.root}>
           {slots.expandIcon && <slots.expandIcon {...slotProps.expandIcon} />}
           {slotProps.root.children}
-          {slots.actions && <slots.actions {...slotProps.actions} />}
+          <ButtonContextProvider value={contextValues.button}>
+            {slots.actions && <slots.actions {...slotProps.actions} />}
+          </ButtonContextProvider>
```

Comment:
For now I'm just wrapping `actions` slot around this context, as it's only `actions` whom might present a button according to design spec. This might be changed in the future to ensure button size on all `TreeItem` component